src/adapters/MeetingsJSONAdapter.test.js

```diff
@@ -1,53 +1,119 @@
+import {from} from 'rxjs';
+import {tap, skip} from 'rxjs/operators';
+
 import meetings from './../data/meetings';
 import MeetingsJSONAdapter from './MeetingsJSONAdapter';
 
 describe('Meetings JSON Adapter', () => {
-  let meetingsJSONAdapter, meetingID;
+  let meetingsJSONAdapter, meetingID, subscription;
 
   beforeEach(() => {
     [meetingID] = Object.keys(meetings);
     meetingsJSONAdapter = new MeetingsJSONAdapter(meetings);
     meetingsJSONAdapter.getLocalVideo = jest.fn(() => Promise.resolve('mock-stream'));
   });
 
-  test('getMeeting() returns an observable', () => {
-    expect(rxjs.isObservable(meetingsJSONAdapter.getMeeting())).toBeTruthy();
+  afterEach(() => {
```

Comment:
we have 2 `afterEaches` now.

Reply:
and honestly, I prefer to have this at the end for *THIS* repo #consistency.

Reply:
Totally didn't even realize the 2 `afterEach`. I combined them at the end of the `describe` block